src/views/StudentDashboard.js

```diff
@@ -102,24 +101,42 @@ export default function StudentDashboard() {
         console.log("err is ", err);
       });
 
-    // const axios_header = {
+    // fetch(`${BACKEND_URL}/stats/student/${student_username}`, {
+    //   method: "POST",
+    //   body: JSON.stringify({}),
     //   headers: {
-    //     Bearer: localStorage.getItem("student_jwt"),
-    //     // "Content-Type": "application/json;charset=UTF-8",
-    //     // "Access-Control-Allow-Origin": "*",
-    //   },
-    // };
-
-    // axios
-    //   .post(`${BACKEND_URL}/stats/student/${student_username}`, axios_header)
+    //     Bearer: `${localStorage.getItem("student_jwt")}`
+    //   }
+    // })
+    //   .then((res) => res.json())
     //   .then((res) => {
     //     console.log(res);
+    //     // setFullName(res.name);
+    //     // setCollegeName(res.college);
+    //     // setEvalStatus(res.evals);
     //   })
     //   .catch((err) => {
-    //     alert("Server error, Try again");
-    //     console.log(err);
+    //     // alert("Server Error, Please try again");
+    //     console.log("err is ", err);
     //   });
 
```

Comment:
Old code should be converted to comments if you think it would be helpful in the future. You can just remove this for now.